documentation/src/main/asciidoc/reference/backend-lucene.asciidoc

```diff
@@ -238,11 +239,12 @@ Multiple strategies are available:
 ----
 # To configure the defaults for all indexes:
 hibernate.search.backend.sharding.strategy = hash
-hibernate.search.backend.sharding.number_of_shards = 2 (no default)
+hibernate.search.backend.sharding.number_of_shards = 2
 # To configure a specific index:
 hibernate.search.backend.indexes.<index name>.sharding.strategy = hash
-hibernate.search.backend.indexes.<index name>.sharding.number_of_shards = 2 (no default)
+hibernate.search.backend.indexes.<index name>.sharding.number_of_shards = 2
 ----
+There is no default for this property.
```

Comment:
I wouldn't include this line: it's confusing, as there are multiple properties listed above.
Also, we already say that `number_of_shards` is required just one line below.

Just remove the `(no default)`.